### PR TITLE
Fix broken More menu when no history action

### PR DESCRIPTION
**Pull request by avoinea**
Refs #2120

### CODE REVIEW

File: cypress/integration/history.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+describe('History Tests', () => {
+  beforeEach(() => {
+    // give a logged in editor and the site root
+    cy.autologin();
+    cy.visit('/');
+    cy.waitForResourceToLoad('@navigation');
+    cy.waitForResourceToLoad('@breadcrumbs');
+    cy.waitForResourceToLoad('@actions');
+    cy.waitForResourceToLoad('@types');
+    cy.waitForResourceToLoad('');
```
Comment (tiberiuichim) [inline cypress/integration/history.js:10]: I wonder if this does anything (I think I've introduced this style when I got rid of ?fullobjects).

Comment (sneridagh) [reply cypress/integration/history.js:10]: I hope it does!

File: src/components/manage/Toolbar/More.jsx [hunk 2]
```diff
@@ -126,30 +126,18 @@ class More extends Component {
               {editAction && <Display pathname={path} />}
             </li>
             <li>
-              {historyAction ? (
```
Comment (sneridagh) [inline src/components/manage/Toolbar/More.jsx:129]: I think we need the history conditional for not showing it in the root, isn't it?

**Review by sneridagh — CHANGES_REQUESTED** [review]
I'll leave the conditional for the root use case, where it seems weird to show it, at least until we have DX-based roots...

File: src/components/manage/Toolbar/More.jsx (hunk 2 above)
Comment (avoinea) [reply src/components/manage/Toolbar/More.jsx:129]: @sneridagh The problem is that in Plone 4 `historyAction` is undefined everywhere, but `/history` is a valid path.

**Comment by sneridagh** [comment]
@avoinea I see, I just checked in the current version and we are showing it at root (I thought we didn't).
Anyways, LGTM!

**Review by sneridagh — APPROVED** [review]

**Pull request merged**